src/romalyzer.cpp

```diff
@@ -2753,18 +2753,15 @@ void ROMAlyzer::on_pushButtonChecksumWizardSearch_clicked()
 	lineEditSets->setEnabled(false);
 	labelStatus->setText(tr("Check-sum search"));
 
-	QString hashStartString;
-	int hashStartOffset;
+    QString hashAttribute;
```

Comment:
This line somehow got indented with spaces rather than a tab.

Reply:
Done